debian/rules

```diff
@@ -1,128 +1,8 @@
+%:
+	dh $@
 
-# Uncomment this to turn on verbose mode.
-#export DH_VERBOSE=1
+DEB_CPPFLAGS_MAINT_APPEND += -DOSSOLOG_COMPILE
+DEB_CFLAGS_MAINT_APPEND   += -Wall -Wmissing-prototypes -Wcast-align
 
-
-# These are used for cross-compiling and for saving the configure script
-# from having to guess our platform (since we know it already)
-DEB_HOST_GNU_TYPE   ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
-DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
-
-
-CFLAGS = -DOSSOLOG_COMPILE -Wall -g \
-         -Wmissing-prototypes -Wcast-align
-# -pedantic-errors -std=c99
-
-# Use soft-float and thumb mode if it is enabled.
-ifneq (,$(findstring thumb,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -mthumb
-endif
-
-CPPFLAGS = -DBONOBO_DISABLE_DEPRECATED \
-           -DBONOBO_UI_DISABLE_DEPRECATED \
-	   -DEEL_DISABLE_DEPRECATED \
-	   -DGCONF_DISABLE_DEPRECATED \
-	   -DG_DISABLE_DEPRECATED \
-	   -DGDK_DISABLE_DEPRECATED \
-	   -DGNOME_DISABLE_DEPRECATED \
-	   -DGNOME_VFS_DISABLE_DEPRECATED \
-	   -DGTK_DISABLE_DEPRECATED \
-	   -DLIBGLADE_DISABLE_DEPRECATED
-
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
-ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
-	INSTALL_PROGRAM += -s
-endif
-
-config.status:
-	./autogen.sh
-	dh_testdir
-	# Add here commands to configure the package.
-	CPPFLAGS="$(CPPFLAGS)" CFLAGS="$(CFLAGS)" ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info
-
-
-build: build-stamp
-
-build-stamp:  config.status
-	dh_testdir
-
-	# Add here commands to compile the package.
-	$(MAKE)
-	#/usr/bin/docbook-to-man debian/osso-app-killer.sgml > osso-app-killer.1
-
-	touch build-stamp
-
-clean:
-	dh_testdir
-	dh_testroot
-	rm -f build-stamp 
-
-	# Add here commands to clean up after the build process.
-	-$(MAKE) distclean
-ifneq "$(wildcard /usr/share/misc/config.sub)" ""
-	cp -f /usr/share/misc/config.sub config.sub
-endif
-ifneq "$(wildcard /usr/share/misc/config.guess)" ""
-	cp -f /usr/share/misc/config.guess config.guess
-endif
-	dh_clean 
-
-install: build
-	dh_testdir
-	dh_testroot
-	dh_clean -k 
-	dh_installdirs
-
-	# Add here commands to install the package into debian/ke-recv.
-	$(MAKE) install DESTDIR=$(CURDIR)/debian/tmp
-	# hack the ke-recv package to work
-	$(MAKE) install DESTDIR=$(CURDIR)/debian/ke-recv
-	rm -rf $(CURDIR)/debian/ke-recv/usr/share/locale
-
-# Build architecture-independent files here.
-binary-indep: build install
-# We have nothing to do by default.
-
-# Build architecture-dependent files here.
-binary-arch: build install
-	dh_testdir
-	dh_testroot
-	dh_installchangelogs ChangeLog
-	dh_installdocs
-#	dh_installexamples
-	dh_install --sourcedir=debian/tmp
-#	dh_installmenu
-#	dh_installdebconf	
-#	dh_installlogrotate
-#	dh_installemacsen
-#	dh_installpam
-#	dh_installmime
-	dh_installinit
-	dh_installinit -u 'start 30 2 3 4 . stop 30 0 1 5 6 .'
```

Comment:
Where is the equivalent in the new build scripts?

Reply:
I missed it.